test/bigtest/interactors/check-in.js

```diff
@@ -86,6 +86,7 @@ import MultiColumnListInteractor from '@folio/stripes-components/lib/MultiColumn
   barcode = fillable('#input-item-barcode');
   clickEnter = clickable('#clickable-add-item');
   checkedInBookTitle = text('#list-items-checked-in div[role="gridcell"]:nth-child(2)');
+  checkedInTimeReturned = text('#list-items-checked-in div[role="gridcell"]:nth-child(1) div div');
```

Comment:
Is there a way to have a more stable selector like giving some data attribute? Imagine the item no longer the first child or the additional div will be added to `div div` would not work.